google/cloud/bigquery/schema.py

```diff
@@ -291,3 +318,72 @@ def _to_schema_fields(schema):
         field if isinstance(field, SchemaField) else SchemaField.from_api_repr(field)
         for field in schema
     ]
+
+class PolicyTagList(object):
+    """Define Policy Tags for a column.
+
+    Args:
+        names (Union[List[str], None]): list of policy tags to associate with
+            the column.
+    """
+
+    def __init__(self, names=None):
+        self._properties = {}
+        if names is not None:
+            self.names = names
+
+    @property
+    def names(self):
+        """Union[List[str], None]: Policy tags associated with this definition.
+        """
+        return self._properties.get("names", ())
```

Comment:
(nit) The default return value is a tuple, not a `List` as the docstrings suggests. Mypy might actually complain about if it detects this. And the setter's doctring also specifies `None` to be the default?